sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/administration/ManageDocumentClassifiersAsync.java

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.administration;
+
+import com.azure.ai.formrecognizer.documentanalysis.administration.DocumentModelAdministrationAsyncClient;
+import com.azure.ai.formrecognizer.documentanalysis.administration.DocumentModelAdministrationClientBuilder;
+import com.azure.core.credential.AzureKeyCredential;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+
+/**
+ * Async sample demonstrating common document classifier management operations on your Form
+ * Recognizer resource.
+ * To learn how to build your own classifiers, look at BuildDocumentClassifierAsync.java and BuildDocumentClassifier.java.
+ */
+public class ManageDocumentClassifiersAsync {
+
+    /**
+     * Main program to invoke the demo.
+     *
+     * @param args Unused. Arguments to the program.
+     */
+    public static void main(final String[] args) {
+        // Instantiate a client that will be used to call the service.
+        DocumentModelAdministrationAsyncClient client = new DocumentModelAdministrationClientBuilder()
+            .credential(new AzureKeyCredential("{key}"))
+            .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+            .buildAsyncClient();
+
+        AtomicReference<String> classifierId = new AtomicReference<>();
+
+        // Next, we get a paged list of all document classifiers
+        System.out.println("We have following classifiers in the account:");
+        client.listDocumentClassifiers().subscribe(documentClassifierDetails -> {
+            System.out.printf("Classifier ID: %s%n", documentClassifierDetails.getClassifierId());
+
+            // get Classifier info
+            classifierId.set(documentClassifierDetails.getClassifierId());
```

Comment:
create a new reusable variable for `documentClassifierDetails.getClassifierId()`